recipes/lammps/gh200/environments.yaml

```diff
@@ -0,0 +1,22 @@
+lammps:
+  compiler:
+  - toolchain: gcc
+    spec: gcc@12.3
+  mpi:
+    spec: cray-mpich@8.1.28
+    gpu: cuda
+  unify: true
+  specs:
+  - cuda@12
+  - kokkos@develop +wrapper
+  - lammps@develop
+  variants:
+  - +mpi
+  - +cuda
+  - cuda_arch=90
+  - +kokkos@develop
```

Comment:
Is this a variant?

Reply:
Do you mean kokkos? It's a variant. See here: https://packages.spack.io/package.html?name=lammps

Reply:
Yes. I think the variant should be +kokkos (without @develop). The way it is written now, I don't think it will be applied. The variants in the "variants:" list are optional (i.e. applied only if they exist), so I think it's better to enable it directly in the lammps spec (unlike +mpi etc which apply to multiple packages).